test/Npgsql.Tests/Replication/CommonReplicationTests.cs

```diff
@@ -21,9 +21,12 @@ public class CommonReplicationTests<TConnection> : SafeReplicationTestBase<TConn
 {
     #region Open
 
-    [Test]
+    [Test, Parallelizable(ParallelScope.None)]
     public async Task Open()
     {
+        // Force type reloading from the replication connection if it is a logical replication connection
+        if (typeof(TConnection) == typeof(LogicalReplicationConnection))
+            Internal.NpgsqlDatabaseInfo.Cache.Clear();
```

Comment:
Do we want to do this on every single connection open in the tests? Just doing it once in the test that exercises composite should be sufficient, no?

Reply:
Oh sorry, I misread.